doc/nrf/known_issues.rst

```diff
@@ -2031,6 +2040,19 @@ NCSDK-14015: Execution halts during boot
 
   **Workaround:** Place the ``rpmsg_nrf53_sram`` partition inside the ``sram_nonsecure`` partition using :ref:`partition_manager`.
 
+<<<<<<< HEAD
+=======
```

Comment:
These should be removed.
```suggestion
```

Reply:
What the... thanks!

Reply:
Forgot to push my edits. Please recheck.

Reply:
LGTM. Thanks!